ktor-network/posix/src/io/ktor/network/selector/SelectableNative.kt

```diff
@@ -1,9 +1,12 @@
 /*
- * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
 package io.ktor.network.selector
 
+import io.ktor.util.*
+
+@KtorExperimentalAPI
```

Comment:
Why did we add an old annotation here?

Reply:
Because it is experimental at the expected side